thumbv7em-tock-eabi.json

```diff
@@ -11,13 +11,7 @@
   "executables": true,
   "no-compiler-rt": false,
   "relocation-model": "ropi-rwpi",
-  "linker": "arm-none-eabi-ld",
-  "linker-flavor": "ld",
+  "linker-flavor": "ld.lld",
   "linker-is-gnu": true,
-  "disable-redzone": true,
-  "pre-link-args": {
```

Comment:
I don't know why but the `pre-link-args` section stopped having any effect on the linker, so I had to move that part to `.cargo/config`.